photon-icons/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "photon-icons",
+  "version": "0.0.1",
+  "description": "Icons for Firefox and related properties.",
+  "scripts": {
+    "build": "node build.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/FirefoxUX/design-tokens.git"
+  },
+  "keywords": [
+    "firefox",
+    "ant",
```

Comment:
This is the only thing that I find odd, probably we could mention tokens or similar instead

Reply:
How about:
``` javascript
  "keywords": [
    "firefox",
    "icons",
    "design",
    "tokens",
    "json"
  ],```